src/pages/Profile.js

```diff
@@ -1,22 +1,49 @@
-import React from 'react';
+import React, {useContext, useEffect, useState} from 'react';
 import { Link } from 'react-router-dom';
+import axios from "axios";
+import { AuthContext } from "../context/AuthContext";
+
 
 function Profile() {
-  return (
-    <>
-      <h1>Profielpagina</h1>
-      <section>
-        <h2>Gegevens</h2>
-        <p><strong>Gebruikersnaam:</strong> hardcoded-test</p>
-        <p><strong>Email:</strong> hardcoded@test.com</p>
-      </section>
-      <section>
-        <h2>Strikt geheime profiel-content</h2>
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab alias cum debitis dolor dolore fuga id molestias qui quo unde?</p>
-      </section>
-      <p>Terug naar de <Link to="/">Homepagina</Link></p>
-    </>
-  );
+    const {...isAuthenticated} = useContext(AuthContext);
```

Comment:
Ik had liever gezien dat je de properties had gespread, dan had je niet overal `isAuthenticated.` vóór hoeven te zetten. En daarnaast doet de naam nu vermoeden dat het een boolean is, terwijl het een object is
```suggestion
    const { username, email } = useContext(AuthContext);
```